Gemfile.lock

```diff
@@ -181,6 +181,9 @@ GEM
       unicode-display_width (>= 1.4.0, < 1.7)
     ruby-progressbar (1.10.1)
     ruby_dep (1.5.0)
+    ruby_http_client (3.3.0)
+    sendgrid-ruby (6.0.4)
+      ruby_http_client (~> 3.3.0)
```

Comment:
Please remove these, I don't think it is required

Reply:
removed